keyboards/3w6/rev2/matrix.c

```diff
@@ -0,0 +1,286 @@
+/*
```

Comment:
Please convert this to matrix lite
https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
Thanks for reviewing the files.

I am a bit lost with this change request. I am under the assumption that it is already using matrix lite. 

rules.mk has CUSTOM_MATRIX = lite
matrix.c is implementing **_matrix_init_custom_** and **_matrix_scan_custom_**

Would you mind giving me a small hint at what I should be looking at?

If there really is a problem, the same will likely exist in the keyboards/3w6/rev1/matrix.c  file, since the only difference in the two is the pins read back in the functions.

Thanks a lot!

Reply:
That's basically it.  It also doesn't need a lot of the additional functions, nor debouncing code, etc. 

For instance, this PR converts the ergodox EZ to the lite version of custom matrix:
https://github.com/qmk/qmk_firmware/commit/11c308d436180974b7719ce78cdffdd83a1302c0#diff-be76b2c6078a2837fdee9d715dbafd5eb34c26fd5ea0911bdf0a942b3b0c5013

Reply:
Thanks a lot for the link.

Sorry for being dense, but I still don't see what the exact problem is. 

The matrix.c file I am using is, with the exception of different addresses/pins, identical to the lite-converted file in the linked ergodox EZ PR.